extensions/Worker.Extensions.ServiceBus/src/Worker.Extensions.ServiceBus.csproj

```
@@ -6,7 +6,7 @@
     <Description>Azure Service Bus extensions for .NET isolated functions</Description>
 
     <!--Version information-->
-    <VersionPrefix>5.22.0</VersionPrefix>
+    <VersionPrefix>5.23.0</VersionPrefix>
```

Are there any new features here, or just the bug fix? If it is a bug fix, rev the patch version (and not minor)

```suggestion
    <VersionPrefix>5.22.1</VersionPrefix>
```

---

👍🏼